assets/src/edit-story/components/panels/videoPoster.js

```diff
@@ -47,11 +51,11 @@ function VideoPosterPanel({ selectedElements, onSetProperties }) {
 
   const handleChangeImage = (image) => {
     const newState = {
-      featuredMedia: image.id,
+      posterId: image.id,
       poster: image.sizes?.medium?.url || image.url,
     };
     setState({ ...state, ...newState });
-    onSetProperties(newState);
+    onSetProperties({ resource: newState });
```

Comment:
Ditto: this will override the whole resource. But I don't think this is a right approach here. The `resource` object would ideally be equivalent (even if not identity equal) across all elements that use it. If a single element overrides the poster, it should be this element's poster and not resource's poster. E.g. the actual used poster for an element is `element.poster || element.resource.poster`.

Reply:
Note this is another issue - https://github.com/google/web-stories-wp/issues/572